docs/configuration.md

```diff
@@ -1180,6 +1200,30 @@ one of the `allow` regular expressions **and** one of the following holds:
 2.  `deny` is set but no URLs within the manifest match any of the `deny` regular
     expressions.
 
+#### `configMediaTypes`
```

Comment:
Could you please document that the white-list and black-list only apply to OCI and schema2 image manifests? schema1, manifest lists, and OCI indicies are not included.